actionview/lib/action_view/helpers/form_tag_helper.rb

```diff
@@ -442,8 +443,14 @@ def text_area_tag(name, content = nil, options = {})
       #   check_box_tag 'eula', 'accepted', false, disabled: true
       #   # => <input disabled="disabled" id="eula" name="eula" type="checkbox" value="accepted" />
       def check_box_tag(name, value = "1", checked = false, options = {})
+        if checked.is_a?(Hash) && options == {}
```

Comment:
```suggestion
        if checked.is_a?(Hash) && options.empty?
```
Nipick: We can save a Hash allocation here.

Reply:
Thanks, I fixed that up.